scripts/jobStaticData.json

```diff
@@ -0,0 +1,23 @@
+{
+    "paladin": {
+        "gauges": {
```

Comment:
@epjenk is this the sort of information you're hoping for? Any format/additions? I'm thinking default assumed cap is 1 so we don't need to go through all the little skill-chains (e.g. GNB grants Ready to Rip, stuff in DRG, etc)